tox.ini

```diff
@@ -19,7 +19,7 @@ passenv =
     HOME
 commands =
     pip install -e .
-    pytest --cov=cookiecutter {posargs:tests}
+    pytest --cov=cookiecutter --cov-report=term --cov-report=html --cov-report=xml {posargs:tests}
```

Comment:
What do you think about moving the reporting to another command so when you use the bare testing to improve the speed of the parallel execution. The coverage is tracked but will be combined in another step.